backend/src/controllers/banner.controller.js

```
@@ -76,14 +121,22 @@ class BannerController {
       if (!position || !closeButtonAction) {
-        return res
-          .status(400)
-          .json({
-            errors: [{ msg: "position and closeButtonAction are required" }],
-          });
+        return res.status(400).json({
+          errors: [{ msg: "position and closeButtonAction are required" }],
+        });
+        return res.status(400).json({
+          errors: [{ msg: "position and closeButtonAction are required" }],
+        });
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)